app/controllers/pair_requests_controller.rb

```diff
@@ -3,8 +3,16 @@ class PairRequestsController < ApplicationController
 
   # GET /pair_requests or /pair_requests.json
   def index
+    if params[:filter] == 'past'
+      @pair_request = PairRequest.new
+      @pair_requests = policy_scope(PairRequest).past.order_by_status.order_by_date
+      render "index", locals: { pair_requests: @pair_requests }
+    else
+      @pair_request = PairRequest.new
+      @pair_requests = policy_scope(PairRequest).upcoming.order_by_status.order_by_date
+    end
```

Comment:
As a small exercise try to see if you can refactor all this to be less repetitive.